RefRed/preview_config/preview_config.py

```diff
@@ -169,8 +164,11 @@ def _retrieve_runs_data(self, _dom):
         _data_table = empty((nbr_row, len(self.data_name)), dtype=object)
         for _row_index, _run_node in enumerate(refl_data):
             for _child_index, _child_name in enumerate(self.data_name):
-                _value = self.get_node_value(_run_node, _child_name)
-                _data_table[_row_index, _child_index] = str(_value)
+                try:
+                    _value = self.get_node_value(_run_node, _child_name)
+                    _data_table[_row_index, _child_index] = str(_value)
+                except:
+                    _data_table[_row_index, _child_index] = ""
```

Comment:
I don't know what data goes here but again, making missing info permissible is a big change. Are there any exceptions to the permissible rule?

Reply:
I think it's useful. The issue here is that the previous version would simply crash when info was missing. I think it's better to load the data so we can inspect it.

Reply:
Also, that would also allow us to preview a file that crashed when loading it for real.